website/serveradmin.php

```diff
@@ -0,0 +1,378 @@
+<?php
```

Comment:
I think there's no reason why this can't be a part of `servers/[servername]`

Plausibly we should have a `website/servers/index.php` and `website/servers/view.php` if we wanted to split things up that way.

Reply:
I feel like this might be better suited to a separate page, still? Say, /servers/manage.php?
Having it integrated into the other pages seems like it would add clutter.

Reply:
You always seem to have a very different definition of clutter than I do. Putting the interface for editing a side server on the page for editing a side server makes more sense than making you dig for it.

Reply:
I mean, this isn't for editing one side server so much as for general administration tasks (banning users from owning, reading server modlogs, banning hosts from being used.) To me, that seems better suited to another page.

Reply:
Oh, I see! Yeah, `servers/manage.php` seems like a good place.

Reply:
Alrighty then.